cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -1213,6 +1255,54 @@ viewHome model packageName packageDef =
 
                         Nothing ->
                             row [ width fill, spacing (Theme.smallSpacing model.theme), padding (Theme.smallPadding model.theme) ] [ text <| "This package contains " ++ String.fromInt numberOfModules ++ " modules." ]
+
+                -- View to display the ValueEditors for Custom Attributes when a node is selected
+                viewAttributeValues : NodeID -> Element Msg
```

Comment:
Since we are using the same code here as further down when displaying the type and value custom attributes, I think this function should be extracted to a top level function and reused in both places

Reply:
This has been done. We now have on version of the viewAttributeValues function